frontend/src/config/constants.ts

```diff
@@ -179,9 +179,22 @@ export const urls = {
 };
 
 export const securityQuestions = [
-  "What’s your first pet’s name?",
-  "What’s your favorite book?",
-  "What’s your favorite food?",
+  "What’s the first name of your best friend in high school?",
```

Comment:
nit: should this be `from high school` instead of `in high school`?

Reply:
Updated in dcf2ebb